web/packages/teleport/src/Account/AccountNew.tsx

```diff
@@ -174,9 +178,9 @@ export function Account({
             header={
               <Header
                 title="Passkeys"
-                description="Passkeys are a password replacement that validates
-                your identity using touch, facial recognition, a device
-                password, or a PIN."
+                description="Enable secure passwordless sign-in using
+                fingerprint or facial identification, a device code, or
```

Comment:
```suggestion
                fingerprint or facial recognition, a one-time code, or
```

I'd stick with _facial recognition_ over _facial identification_.